src/v/kafka/server/tests/create_topics_test.cc

```diff
@@ -96,7 +96,6 @@ class create_topic_fixture
 
     void test_create_topic(
       kafka::create_topics_request req,
```

Comment:
I see that you are passing the partition_count now to the create_topic function, but the partition_count that I passed here was the partition_count of the original topic. So the idea is that you have a manifest of the original topic with partition_count = 32, create a read_replica topic with partition_count=1, but when you verify metadata of  a read replica topic, you should see 32. Value from manifest should override value passed in create topic request

Reply:
gotcha makes more sense to me now, i will just revert this change